src/Components/Shifting/ListView.tsx

```diff
@@ -60,171 +61,181 @@ export default function ListView() {
   });
 
   const showShiftingCardList = (data: any) => {
-    if (data && !data.length) {
+    if (!data || data.length === 0) {
```

Comment:
When `data` from useQuery is undefined, its likely that it's still loading. Show `<Loading />` in such situations. Can you confirm the same and show Loading if so?

Reply:
Ok, I'll add that